2017-bae-europe/eu_future/eu_future.xml

```diff
@@ -6054,8 +5633,54 @@
     </institution>
   </region>
 
-
-<!-- Recipes -->
+  
+<region>
+    <name>Sink_Source</name>
```

Comment:
I just got rid of the region and placed the institution under France Region
as `french_fuel_facilities` institution.